.github/workflows/cypress_workflow.yml

```diff
@@ -49,7 +51,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+          group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
```

Comment:
can you link FTR repo change for ci-group 11

Reply:
Hi this PR is old, i have a new PR #8703 for the changes.